deploy/README.md

```diff
@@ -44,4 +44,14 @@ To see all resources installed by the helm chart
 
 ```sh
 kubectl get elastic -l "app.kubernetes.io/instance"=es-kb-quickstart -n elastic-stack
-```
\ No newline at end of file
+```
+
+## Licensing
+
+### ECK Operator Helm Chart
+
+The ECK Operator is licensed under the [Elastic License 2.0](https://www.elastic.co/licensing/elastic-license), and is free to use under the limitations defined within the License.
```

Comment:
This may be confusing to some.
```suggestion
The ECK Operator and its Helm Chart are licensed under the [Elastic License 2.0](https://www.elastic.co/licensing/elastic-license), and are free to use under the limitations defined within the License.
```